modules/installation-gcp-shared-vpc-ingress.adoc

```diff
@@ -0,0 +1,49 @@
+// File included in the following assemblies:
+// * installation/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-ingress_{context}"]
+= Optional: Adding ingress DNS records for shared VPC installations
+If the public DNS zone exists in a host project outside the project where you installed your cluster, you must manually create DNS records that point at the ingress load balancer. You can create either a wildcard `*.apps.{baseDomain}.` or specific records. You can use A, CNAME, and other records per your requirements.
```

Comment:
Does a period actually go after {baseDomain}. ? I'm assuming yes, but just making sure.

Reply:
Yup, it sure does.